src/main/java/com/greencity/ui/page/homepage/HomePage.java

```diff
@@ -2,10 +2,269 @@
 
 import com.greencity.ui.page.BasePage;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
 
 public class HomePage extends BasePage {
+    private WebDriver driver;
```

Comment:
removed this line `private WebDriver driver;` because you are inheriting it from `BasePage`